trl/commands/scripts

```diff
@@ -0,0 +1 @@
+/home/azureuser/caf83/trl/examples/scripts/
```

Comment:
please remove this file

Reply:
Sorry, this slipped in there when adding changed files. Good catch